samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml

```diff
@@ -193,6 +197,10 @@ spec:
       defaults:
         - docker.io
         - quay.io
+    - name: storage-driver
+      description: The storage driver for buildah
```

Comment:
In text form, can we document the possible values, maybe a Link to the BuildAh documentation with guidance on how to set this?

Reply:
+1

Reply:
Updated the details. PTAL. Thanks!